airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/EnrollmentModal/EnrollmentModal.tsx

```diff
@@ -57,6 +64,32 @@ export const EnrollmentModalContent: React.FC<EnrollmentModalContentProps> = ({
     };
   }, []);
 
+  const EnrollmentCta = () =>
+    emailVerified ? (
+      <Button isLoading={isLoading} onClick={startStripeCheckout}>
+        <FormattedMessage id="freeConnectorProgram.enrollmentModal.enrollButtonText" />
+      </Button>
+    ) : (
+      <Button isLoading={isLoading} onClick={sendEmailVerification}>
+        <FormattedMessage id="freeConnectorProgram.enrollmentModal.unvalidatedEmailButtonText" />
+      </Button>
+    );
+
+  const EnrollmentEmailVerificationWarning = () => {
+    const WarningContent = () => (
+      <FlexContainer>
+        <FlexContainer justifyContent="center" className={styles.iconContainer}>
+          <MailSVG />
+        </FlexContainer>
+        <Text size="lg" className={styles.warning}>
+          <FormattedMessage id="freeConnectorProgram.enrollmentModal.unvalidatedEmailWarning" />
+        </Text>
+      </FlexContainer>
```

Comment:
I think the `<Callout/>` mentioned in Slack would be less visually harsh than the red text here.

Reply:
Agreed, definitely nicer.